Makefile

```diff
@@ -0,0 +1,69 @@
+# Define the Python command
```

Comment:
I am not a fan of Makefile for Python code. Also, the new method for creating and updating PO files for all the tools written in Python must be unified. It would be better to change this with setup.py.

See https://github.com/ghostbsd/issues/issues/196.